src/EditorFeatures/VisualBasicTest/CodeLens/VisualBasicCodeLensTests.vb

```diff
@@ -0,0 +1,137 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Threading
+Imports Microsoft.CodeAnalysis.CodeLens
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.CodeLens
+    Public Class VisualBasicCodeLensTests
+        Private _referenceService As CodeLensReferenceService
+
+        Private Async Function SetupWorkspaceAsync(content As String) As Task(Of TestWorkspace)
+            Dim workspace = Await TestWorkspace.CreateVisualBasicAsync(content)
+            _referenceService = New CodeLensReferenceService()
+            Return workspace
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeLens)>
+        Public Async Function TestCount() As Task
+            Using workspace = Await SetupWorkspaceAsync("" + vbCrLf +
```

Comment:
Yes.  See our FindReferences tests.  We share a single test driver for both languages.  And all the tests are declarative instead of being imperatively written.